hack/jenkins/verify-jenkins-jobs.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
it'd be a good idea to set errexit, nounset, and pipefail at least. (xtrace can sometimes be helpful too, though not strictly necessary)